strawberry/django/views.py

```diff
@@ -87,13 +88,31 @@ def _render_graphiql(self, request: HttpRequest, context=None):
 
         return response
 
+    def _create_response(
+        self, response_data: GraphQLHTTPResponse, sub_response: HttpResponse
+    ) -> JsonResponse:
+        response = JsonResponse(response_data)
+        response._headers.update(sub_response._headers)
+
+        # unfortunately setting None as status code doesn't work
+        # so to make sure we only override status codes when they have been
+        # changed we use a status code that doesn't actually exist
+        # https://github.com/django/django/blob/03d0f12/django/http/response.py#L117-L124
+        if sub_response.status_code != 199:
```

Comment:
Do you think if it's worth using some private constant for this integer?

Reply:
that's a good suggestion, we could also:

1. have a subclass of httpresponse that allows to set status_code to none (or uses None as default)
2. set status_code as none after creating the instance

let me try 1

Reply:
@alex-verve 1 worked well 😊